### PR TITLE
Update changelog for v7.0.0-beta1

### PR DESCRIPTION
**Pull request by marefr**
**What this PR does / why we need it**:
Update changelog for v7.0.0-beta1

**Special notes for your reviewer**:
This changelog is generated based on PR's and issues in 7.0-beta1 milestone having "add to changelog" label. I have already made some changes to PR titles.

Please have a look and add suggestions on things you want to change and/or make changes and push them.

### CODE REVIEW

**Review by aknuds1 — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,168 @@
...
 - **Dashboard**: A global minimum dashboard refresh interval is now enforced and defaults to 5 seconds.
 
+# 7.0.0-beta1 (2020-04-28)
+
+## Breaking changes
+
+- **Removed PhantomJS**: PhantomJS was deprecated in [Grafana v6.4](https://grafana.com/docs/grafana/latest/guides/whats-new-in-v6-4/#phantomjs-deprecation) and starting from Grafana v7.0.0, all PhantomJS support has been removed. This means that Grafana no longer ships with a built-in image renderer, and we advise you to install the [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer).
+- **Docker**: Our Ubuntu based images have been upgraded to Ubuntu [20.04 LTS](https://releases.ubuntu.com/20.04/).
+- **Dashboard**: A global minimum dashboard refresh interval is now enforced and defaults to 5 seconds.
+### @grafana/ui forms migration notice
```
Comment (marefr) [inline CHANGELOG.md:16]: @tskarhed what do you think if we add a link from here to grafana/ui changelog?

Comment (tskarhed) [reply CHANGELOG.md:16]: Yeah, that would work too

Comment (marefr) [reply CHANGELOG.md:16]:
```suggestion
- **@grafana/ui**: Forms migration notice, see [@grafana/ui changelog](https://github.com/grafana/grafana/blob/master/packages/grafana-ui/CHANGELOG.md)

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,168 @@
...
+```
+To this:
+```jsx
+import { Switch} from '@grafana/ui';
+…
+<Switch ... />
+```
+
+To see the new form components visit [our Storybook](https://developers.grafana.com/ui)
+
```
Comment (marefr) [inline CHANGELOG.md:61]:
```suggestion

```

File: CHANGELOG.md (hunk 1 above)
Comment (marefr) [reply CHANGELOG.md:16]: @tskarhed added suggestions

**Review by daniellee — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,123 @@
...
+* **Dashboard**: Support data links via field overrides. [#23590](https://github.com/grafana/grafana/pull/23590), [@dprokop](https://github.com/dprokop)
+* **Data source**: Max data points now used in interval calculation for all data sources. [#23915](https://github.com/grafana/grafana/pull/23915), [@torkelo](https://github.com/torkelo)
+* **Database**: Order results in UserSearch by username/email. [#23328](https://github.com/grafana/grafana/pull/23328), [@aknuds1](https://github.com/aknuds1)
+* **Database**: Update the xorm dependency to v0.8.1. [#22376](https://github.com/grafana/grafana/pull/22376), [@novalagung](https://github.com/novalagung)
+* **Docker**: Upgrade to Ubuntu 20.04 in Dockerfiles. [#23852](https://github.com/grafana/grafana/pull/23852), [@aknuds1](https://github.com/aknuds1)
+* **Docs**: Adding API reference documentation support for the packages libraries. [#21931](https://github.com/grafana/grafana/pull/21931), [@mckn](https://github.com/mckn)
+* **Explore**: Add trace UI to show traces from tracing datasources. [#23047](https://github.com/grafana/grafana/pull/23047), [@aocenas](https://github.com/aocenas)
```
Comment (aocenas) [inline CHANGELOG.md:50]:
```suggestion
* **Explore**: Add trace UI to show traces from tracing datasources and Jaeger datasource. [#23047](https://github.com/grafana/grafana/pull/23047), [@aocenas](https://github.com/aocenas)
```

Comment (marefr) [reply CHANGELOG.md:50]: @aocenas not sure that suggestion makes sense. You've added ZipKin datasource support as well, right?

Comment (aocenas) [reply CHANGELOG.md:50]: Yes but zipkin was separate PR. Daniel was confused why jaeger was not in the changelog and the reason is it was added in this PR

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,123 @@
...
+* **Stackdriver**: Support for SLO queries. [#22917](https://github.com/grafana/grafana/pull/22917), [@sunker](https://github.com/sunker)
+* **Tabel**: Add support for organizing fields/columns. [#23135](https://github.com/grafana/grafana/pull/23135), [@mckn](https://github.com/mckn)
+* **Table**: Improvements to column resizing, style and alignment. [#23663](https://github.com/grafana/grafana/pull/23663), [@torkelo](https://github.com/torkelo)
+* **Table**: upgrades react-table to 7.0.0 and typings. [#23247](https://github.com/grafana/grafana/pull/23247), [@hugohaggmark](https://github.com/hugohaggmark)
+* **TablePanel**: Handle column overflow and horizontal scrolling in table panel. [#4157](https://github.com/grafana/grafana/issues/4157)
+* **Tracing**: Dark theme styling for TraceView. [#23406](https://github.com/grafana/grafana/pull/23406), [@aocenas](https://github.com/aocenas)
+* **Tracing**: Zipkin datasource. [#23829](https://github.com/grafana/grafana/pull/23829), [@aocenas](https://github.com/aocenas)
+* **Transformations**: Adds labels as columns transformer. [#23703](https://github.com/grafana/grafana/pull/23703), [@hugohaggmark](https://github.com/hugohaggmark)
```
Comment (hugohaggmark) [inline CHANGELOG.md:100]:
```suggestion
* **Transformations**: Adds labels as fields transformer. [#23703](https://github.com/grafana/grafana/pull/23703), [@hugohaggmark](https://github.com/hugohaggmark)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,123 @@
...
+* **Provisioning**: Allows specifying uid for datasource and use that in derived fields. [#23585](https://github.com/grafana/grafana/pull/23585), [@aocenas](https://github.com/aocenas)
+* **Provisioning**: Validate that dashboard providers have unique names. [#22898](https://github.com/grafana/grafana/pull/22898), [@youshy](https://github.com/youshy)
+* **Search**: Replace search implementation. [#23855](https://github.com/grafana/grafana/pull/23855), [@sakjur](https://github.com/sakjur)
+* **Search**: migrate dashboard search to react. [#23274](https://github.com/grafana/grafana/pull/23274), [@Clarity-89](https://github.com/Clarity-89)
+* **Server**: Don't include trailing slash in cookie path when hosting Grafana in a sub path. [#22265](https://github.com/grafana/grafana/pull/22265), [@consideRatio](https://github.com/consideRatio)
+* **Stackdriver**: Support for SLO queries. [#22917](https://github.com/grafana/grafana/pull/22917), [@sunker](https://github.com/sunker)
+* **Tabel**: Add support for organizing fields/columns. [#23135](https://github.com/grafana/grafana/pull/23135), [@mckn](https://github.com/mckn)
```
Comment (hugohaggmark) [inline CHANGELOG.md:94]:
```suggestion
* **Table**: Add support for organizing fields/columns. [#23135](https://github.com/grafana/grafana/pull/23135), [@mckn](https://github.com/mckn)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,6 +6,123 @@
...
+* **Table**: Improvements to column resizing, style and alignment. [#23663](https://github.com/grafana/grafana/pull/23663), [@torkelo](https://github.com/torkelo)
+* **Table**: upgrades react-table to 7.0.0 and typings. [#23247](https://github.com/grafana/grafana/pull/23247), [@hugohaggmark](https://github.com/hugohaggmark)
+* **TablePanel**: Handle column overflow and horizontal scrolling in table panel. [#4157](https://github.com/grafana/grafana/issues/4157)
```
Comment (hugohaggmark) [inline CHANGELOG.md:97]:
```suggestion
* **Table**: Handle column overflow and horizontal scrolling in table panel. [#4157](https://github.com/grafana/grafana/issues/4157)
```

**Pull request merged**